R/relativeimportance.R

```diff
@@ -347,13 +349,15 @@ computeJaccardCoefficientOutput <- function(formula, data = NULL, weights, varia
     test.statistics <- vapply(test.output, "[[", numeric(1), "t")
     relative.importance <- 100 * prop.table(abs(test.statistics))
     pvalues <- vapply(test.output, "[[", numeric(1), "p.value")
+    pvalues <- pvalAdjust(pvalues, correction)
     standard.errors <- vapply(test.output, "[[", numeric(1), "standard.error")
     names(pvalues) <- names(standard.errors) <- variable.names
     sample.size <- vapply(X, function(x) sum(!is.na(x) & !is.na(y)), numeric(1))
     names(sample.size) <- variable.names
     list(importance = relative.importance,
-         raw.importance.score = jaccard.coefs,
+         raw.importance = jaccard.coefs,
```

Comment:
is `raw.importance.score` used anywhere else in the code that needs to be updated? e.g. flipFormat

Reply:
Not used anywhere else. Compatible with `flipFormat` due to the partial matching with `raw.importance`. Reverted `flipFormat` to use `raw.importance`  instead in https://github.com/Displayr/flipFormat/commit/2933fb0df8487d812994bc820967a09306c765ff